common.lic

```diff
@@ -197,9 +197,14 @@ module DRC
   module_function :get_noun
 
   def empty_hands
```

Comment:
Only in favor, I believe, which doesn't currently depend on the equipment
manager. I'm hesitant to add it since I imagine favor being an obvious
thing for someone completely new to the project to invoke.

On Tue, Jan 12, 2016 at 2:06 PM, Robert Herbig <notifications@github.com>
wrote:

> In common.lic
> <https://github.com/rpherbig/dr-scripts/pull/174#discussion_r49506115>:
>
> > @@ -197,9 +197,14 @@ module DRC
> >    module_function :get_noun
> >
> >    def empty_hands
>
> Where is this method used now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/174/files#r49506115>.
>